src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/Metadata/IModelBindingMessageProvider.cs

```diff
@@ -25,6 +25,13 @@ public interface IModelBindingMessageProvider
         Func<string> MissingKeyOrValueAccessor { get; }
 
         /// <summary>
+        /// Error message the model binding system adds when no value is provided for the request body,
+        /// but a value is required.
+        /// </summary>
+        /// <value>Default <see cref="string"/> is "A value for the request body was not provided.".</value>
+        Func<string> MissingRequestBodyRequiredValueAccessor { get; }
```

Comment:
@Eilon do you want to avoid a breaking change here too? Could reuse `MissingBindRequiredValueAccessor`. That message (by default `A value for the '{0}' property was not provided.`) would be correct but doesn't mention the request body.